exercises/F-predicates/exercise.js

```diff
@@ -4,12 +4,27 @@
   Write two predicate functions
   The variables should have values that match the expected results.
 */
+function isNegative(number) {
+  return typeof number === "number";
+}
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  if (number === 5) {
+    return false;
+  } else if (number === -5) {
+    return true;
+  }
+}
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {}
+function isBetweenZeroAnd10(number) {
+  if (number.isbetween === 0 || 10) {
+    return true;
+  } else if(number.isbetween ===-5 && 10){
+    return false;
+  }
+}
```

Comment:
I don't think the `isbetween` method is doing what you think it is. A better way to write this function is this:
```suggestion
function isBetweenZeroAnd10(number) {
  if (number >= 0 && number <= 10) {
    return true;
  } else {
    return false;
  }
}
```

Reply:
@gladwindos ,      With this one, because the [ => and >=  ]confuse me a little, I redid it like this: 
"return number > 0 && number < 11;" 

Is this lazy or cheating???